meetings/2021-07-28.md

```diff
@@ -0,0 +1,136 @@
+The first thing we have to do is how to build apps that work accross pods?
+Everyone is using social networks because it's easy to use.
+If we can get everybody to be working on decentralised apps, we're solving a real problem. Not how does the CIA deals with it?
+For the moment, the problem is how does an app following links knows how to authenticate?
+Currently with WAC, it can't find out.
+Because right now, you follow a link and the only person that follows the link and wants to read the ACL is the controller. There is no other way of saying it.
+If I am a friend of yours and follow links and want to read something, I have no idea what identity to use in order to access resources.
+This is not a decentralised network, it is 1990s PC.
+There are a ton of experiments that put a user at the center of using a cloud based system.
+That is not decentralisation.
+We can't make the same mistake.
+The question is how do I comming accross a link what the access control rules are.
+There is a simple answer, which would be making the ACLs more readable than the resources.
+
+Sarven: We can introduce a new access mode like ControlRead. Or using problem details eg. on 403, payload indicating required credentials (over 18), we can describe the structure/semantics as well on the Protocol level.
+
+Henry: We need the access control rules to be readable by the client.
+
+Elf: I don't think that's correct. I strongly disagree with the statement you make that it is holding in all cases.
+
+Henry: OK, but even me with my client that is open source, I should be able to edit my own content that I want to see.
+
+Elf: I login to my application and the application will use the identity I use for logging in.
+
+Henry: It's a very simple answer, all IDs are currently local to the server you're on. Your IDs are tied to a domain. and the browser vendors don't have the imagination to address that.
+
+Elf: What we're discussing in the interop panel is that you would notify agents that they have access to x. So applications have a discovery mechanism and the agent would know which identity is tied to access to specific resources.
+
+Matthieu: Is it a centralised cross identities notification system?
+
+Elf: At the moment we don't define how to keep track of all your identities.
+But each identity would have an associated inbox where they receive the notification about what they have access to.
+The authorization agent of Alice would have access to specific grants and not all applications would have access to all grants.
+
+Matthieu: I don't understand why the default from a security perspective is not 404.
+
+Sarven: First of all, 403 is defined and has meaning. If you can read the container and see that foo is linked to it, you already know that the resource exists. If you get that resource, you can get a 403, but since you know the resource exists 404 doesn't make sense. But the RFC also says you can use 404 to hide information.
```

Comment:
```suggestion
Sarven: First of all, `403` is defined and has meaning. If you can read the container and see that `foo` is linked to it, you already know that the resource exists. If you get that resource, you can get a `403`, but since you know the resource exists `404` doesn't make sense. But the RFC also says you can use `404` to hide information.
```